t/model/DeviceLocation.t

```diff
@@ -2,12 +2,11 @@ use Mojo::Base -strict;
 use Test::More;
 use Test::ConchTmpDB qw(mk_tmp_db);
 
-use IO::All;
-
 use_ok("Conch::Model::DeviceLocation");
 
 use Data::UUID;
 use Conch::Pg;
+use Path::Tiny;
```

Comment:
You don't actually use `Path::Tiny` here.

Reply:
ah yes. I was, and then I removed the sql loading in this file because it's useless.